frontend/src/app/oe/components/oe-master-page/oe-master-page.component.scss

```diff
@@ -60,6 +60,14 @@ li.nav-item {
       font-size: 1em;
     }
   }
+
+  &.account-token-popup {
+    bottom: 0px;
+
+    @media (max-width: 420px) {
+      bottom: 56px;
```

Comment:
we should consider using relative lengths like `em` or `rem` so as per the screen pixel density - its adapts?

Reply:
we might do this later. Currently, we are just focusing on MVP.

Reply:
ok